src/libsystemd/sd-device/sd-device.c

```diff
@@ -561,7 +561,6 @@ int device_read_uevent_file(sd_device *device) {
 
                         state = VALUE;
 
-                        break;
```

Comment:
I'd really prefer if we'd add a comment like `/* fall through */` or so in cases like this, which look like a missing "break" bug, but aren't.